### PR TITLE
pushing changes to branch

### PR DESCRIPTION
**Pull request by atvick21**
Just pushing my changes to the registered users classes. I am not quite done changing things around.

### CODE REVIEW

**Review by dnguyen000 — APPROVED** [review]

**Review by f947 — APPROVED** [review]

File: src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java [hunk 1]
```diff
@@ -4,34 +4,45 @@
...
+	@GetMapping("/all")
+	public List<RegisteredUser> getUsers() {
+		return userService.findAll();
 	}
 	
 	@GetMapping(value = "{userId}")
-	public RegisteredUser getUserById(String id) {
-		return this.registeredUserRepo.findByUserId(id);
+	public RegisteredUser getUserById(@PathVariable String id) {
+		return this.userService.findByUserId(id);
```
Comment (calebTree) [inline src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java:35]: this.userService.getUserById(is);

File: src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java [hunk 2]
```diff
@@ -4,34 +4,45 @@
...
+	}
+	
+	@GetMapping(value = "/email")
+	public RegisteredUser getUserByEmail(@PathVariable String email) {
+		return this.userService.findByUserEmail(email);
```
Comment (calebTree) [inline src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java:40]: `return this.userService.getByUserEmailAndPassword(email);

File: src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java [hunk 3]
```diff
@@ -4,34 +4,45 @@
...
-	@GetMapping(value = "/all")
-	public List<RegisteredUser> list() {
-		return registeredUserRepo.findAll();
+	public RegisteredUserController(RegisteredUserService userService) {
+		this.userService = userService;
+	}
+	
+	@GetMapping("/all")
+	public List<RegisteredUser> getUsers() {
+		return userService.findAll();
```
Comment (calebTree) [inline src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java:30]: `return userServices.getUsers();`

File: src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java [hunk 4]
```diff
@@ -4,34 +4,45 @@
...
+	
+	@GetMapping(value = "/email")
+	public RegisteredUser getUserByEmail(@PathVariable String email) {
+		return this.userService.findByUserEmail(email);
 	}
 	
 	@PostMapping("/add")
 	public RegisteredUser create(@RequestBody final RegisteredUser registeredUser) {
-		return registeredUserRepo.save(registeredUser);
+		return userService.save(registeredUser);
```
Comment (calebTree) [inline src/main/java/edu/metrostate/ics499/team2/controllers/RegisteredUserController.java:45]: `return userService.create(storedUser);

**Review by calebTree — APPROVED** [review]

**Review by wale0 — APPROVED** [review]

**Pull request merged**